src/test/scala/chiselTests/SwitchSpec.scala

```diff
@@ -32,4 +32,22 @@ class SwitchSpec extends ChiselFlatSpec with Utils {
       })
     }
   }
+  it should "provide useful source locators" in {
+    val verilog = ChiselStage.emitVerilog(new Module {
```

Comment:
No need to go to Verilog here. Just `emitChirrtl` should be sufficient to check the source locator strings. It's probably worth changing the val name to not mention "verilog" then.